### DISCUSSION TITLE
Pivot from Sensors interface to Grafana

### DISCUSSION THREAD

**Discussion by darth-drew** [Security-Onion-Solutions/securityonion#1966] [Ideas] [answered]
For our sysadmins spinning up new nodes, it would be cool for them to be able to click on a new line in the sensors interface (similar to the chevron in the alerts interface) and jump to the corresponding grafana dashboard, in order to quickly check on health and status.

**Comment by dougburks** [comment, marked as answer]
Thanks for the idea. We will take this into consideration.

**Reply by darth-drew** [reply]
Trying to bring to the top again for consideration. The more we scale out, the more useful this would be.

**Reply by dougburks** [reply]
We are planning something like this for Security Onion 2.4.

**Reply by OzzyKampha** [reply]
Is there a eta for 2.4?

**Reply by dougburks** [reply]
We plan to share more information about 2.4 at our conference next week:
https://securityonion.net/conference

**Reply by dougburks** [reply]
We shared more information about Security Onion 2.4 at our conference last week and mentioned this specific feature:
https://blog.securityonion.net/2022/10/sneak-peek-at-security-onion-24.html

**Reply by darth-drew** [reply]
I saw! Looking forward to it! Great work!
